src/commands/dev/lib/types.ts

```diff
@@ -0,0 +1,129 @@
+import http from 'http';
+import { Lambda } from '@zeit/fun';
+import { LambdaRuntime } from '@now/build-utils';
+import FileFsRef from '@now/build-utils/file-fs-ref';
+
+export enum DevServerStatus {
+  busy,
+  idle,
+  error
+}
+
+export interface DevServerOptions {
+  debug: boolean;
+}
+
+export interface EnvConfig {
+  [name: string]: string;
+}
+
+export interface BuildConfig {
+  src: string;
+  use: string;
+  config?: object;
+  builder?: Builder;
+}
+
+export interface RouteConfig {
+  src: string;
+  dest: string;
+  methods?: string[];
+  headers?: HttpHeadersConfig;
+  status?: number;
+}
+
+export interface NowConfig {
+  name?: string;
+  version?: number;
+  env?: EnvConfig;
+  build?: {
+    env?: EnvConfig;
+  };
+  builds?: BuildConfig[];
+  routes?: RouteConfig[];
+}
+
+export interface HttpHandler {
+  (req: http.IncomingMessage, res: http.ServerResponse): void;
+}
+
+export interface BuilderInputs {
+  [path: string]: FileFsRef;
+}
+
+export interface BuiltFileFsRef extends FileFsRef {
+  buildConfig?: BuildConfig;
+}
+
+export type BuilderOutput = BuiltLambda | BuiltFileFsRef;
+
+export interface BuilderOutputs {
+  [path: string]: BuilderOutput;
+}
+
+export interface BuilderParamsBase {
+  files: BuilderInputs;
+  entrypoint: string;
+  config: object;
+  isDev?: boolean;
+}
+
+export interface BuilderParams extends BuilderParamsBase {
+  workPath: string;
+}
+
+export interface PrepareCacheParams extends BuilderParamsBase {
+  cachePath: string;
+}
+
+export interface Builder {
+  config?: {
+    maxLambdaSize: string | number;
```

Comment:
```suggestion
    maxLambdaSize?: string | number;
```

I think this config is optional so you want to make sure it exists before reading it.

Reply:
What happens if it is not optional ?

Reply:
bwt **maxLambdaSize** is having an error, apparently it is not requested now-dev when dependencies are installed that require more than **5mb** 🤔